src/main/scala/palefat/data/mirror/Runner.scala

```diff
@@ -77,7 +77,7 @@ object Runner extends LogSupport {
         FilterBuilder.buildReplaceWherePredicate(
           ds,
           config.lastPartitionCol,
```

Comment:
Is it right that Z-ordering is executed using `partitionCols.last`?

Reply:
this is used to filter data. and yes, if it's not a partition column, it's failing with error #15 
z-order itself is executed on a column specified in zorderby_col, a partition is used only to run in on a subset of data